examples/README.md

```diff
@@ -311,6 +311,7 @@ Example | Description
 [Font Atlas Debug](../examples/ui/font_atlas_debug.rs) | Illustrates how FontAtlases are populated (used to optimize text rendering internally)
 [Text](../examples/ui/text.rs) | Illustrates creating and updating text
 [Text Debug](../examples/ui/text_debug.rs) | An example for debugging text layout
+[Toolbar](../examples/ui/toolbar.rs) | Showcases a more advanced UI with conditional rendering, events and managed state
```

Comment:
Nit: I forget Bevy's stance on this (so maybe someone will correct me), but maybe add an Oxford comma? (Same for Cargo.toml)
```suggestion
[Toolbar](../examples/ui/toolbar.rs) | Showcases a more advanced UI with conditional rendering, events, and managed state
```